docs/manual/guides/local-installation/ddev.en.md

```diff
@@ -26,96 +26,71 @@ more information about installing these programs.
 
-### Example: Installation on Debian/Ubuntu
-
-```shell
-curl -fsSL https://apt.fury.io/drud/gpg.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/ddev.gpg > /dev/null
-
-echo "deb [signed-by=/etc/apt/trusted.gpg.d/ddev.gpg] https://apt.fury.io/drud/ * *" | sudo tee /etc/apt/sources.list.d/ddev.list
-
-sudo apt update && sudo apt install -y ddev
+```env
+APP_ENV=prod
+DATABASE_URL=mysql://db:db@db:3306/db
+MAILER_DSN=smtp://localhost:1025
 ```
 
-If necessary, install the local SSL certificates after the installation (afterwards restart your browser).
+Then create the database:
 
 ```shell
-mkcert -install
+ddev exec "bin/console contao:migrate"
 ```
 
-Installation Update
+Create backend user:
 
 ```shell
-sudo apt update && sudo apt upgrade
+ddev exec "bin/console contao:user:create"
 ```
 
-
-## Create project
-
-Open the console of your choice, create the desired directory and then change to it.
-
-```shell
-mkdir -p ~/projects/contao/contao-ddev && cd ~/projects/contao/contao-ddev
-```
-
-Create the DDEV configuration with:
+To use Apache instead of NGINX, change your entry `webserver_type: nginx-fpm` to `apache-fpm`.
 
 ```shell
-ddev config
+ddev launch
 ```
 
-Make DDEV settings, select `php` as __Project Type__ in any case. Leave the __Docroot Location__ empty for now, because there is no `public` folder for new installations and DDEV can not start then.
-
-```shell
-ddev start
-```
+{{% notice note %}}
 
-For installation via console it is easiest to go via ssh into the container.
+With `ddev launch contao` you get directly to the administration.
 
-```shell
-ddev ssh
-```
+{{% /notice %}}
 
-```shell
-composer create-project contao/managed-edition contao 4.13
-```
+The `ddev` binary is not available in the container, so first switch to the host console with `exit`.
 
-In the `.ddev/config.yml` now adjust the docroot and restart ddev.
+## Additional information
 
-```yml
-docroot: "contao/public"
-```
+`ddev start` starts the project, `ddev stop` ends it. Make sure beforehand that you have changed to the project folder.
 
-To use Apache instead of NGINX, change your entry `webserver_type: nginx-fpm` to `apache-fpm`.
+`ddev poweroff` can stop all started projects/containers from any directory.
 
-```yml
-webserver_type: apache-fpm
-```
+With `ddev ssh` you can switch to the shell of the container and work on the console. The `ddev` binary is not available in the container, so first switch to the host console with `exit`.
 
-The `ddev` binary is not available in the container, so first switch to the host console with `exit`.
+`ddev describe` gives an overview of the services available in the project and how to access them.
 
-```shell
-ddev restart
-```
+`ddev xdebug on` starts Xdedug. [Information about the IDE setup](https://ddev.readthedocs.io/en/latest/users/debugging-profiling/step-debugging/#ide-setup)
 
 A database already exists in DDEV. The connection data for the installation are:
```

Comment:
I think this could be removed here (see: DATABASE_URL=mysql://db:db@db:3306/db )

Reply:
removed